src/main/resources/upgrade-files/post-upgrade/sql/UAF-3618_add_parm_values_to_DB_upgrade.sql

```diff
@@ -35,4 +35,16 @@ WHERE NMSPC_CD = 'KUALI-TAX';
 
 UPDATE KRIM_PERM_ATTR_DATA_T
 SET ATTR_VAL='KFS-TAX'
-WHERE ATTR_VAL='KUALI-TAX';
\ No newline at end of file
+WHERE ATTR_VAL='KUALI-TAX';
+
+UPDATE KRIM_PERM_ATTR_DATA_T
+SET ATTR_VAL='edu.arizona.kfs.module.purap.document.web.struts '
```

Comment:
Should be:
SET ATTR_VAL='edu.arizona.kfs.module.tax.document.web.struts.ElectronicFilingReportAction'
The attribute type of this permission attribute is "actionClass", so the actual class name is needed. Also, this is one of the classes that is being moved to the new kfs-tax optional module.